dspace/config/spring/api/discovery.xml

```diff
@@ -53,8 +54,15 @@
                <entry key="site" value-ref="homepageConfiguration" />
                <!--<entry key="123456789/7621" value-ref="defaultConfiguration"/>-->
                <!-- Used to show filters and results on MyDSpace -->
+               <!-- Do not change the id of special entries or else they won't work -->
+               <!-- "workspace" is a special entry to search for your own workspac
+
+               e items -->
```

Comment:
There's a weird gap in this comment.  Tiny thing to cleanup.

Reply:
This has been fixed